plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/TestDeltaLakeMetastoreAccessOperations.java

```diff
@@ -278,6 +283,143 @@ public void testDropTable()
                         .build());
     }
 
+    @Test
+    public void testInformationSchemaColumns()
+    {
+        for (int tables : ImmutableList.of(3, MAX_PREFIXES_COUNT, MAX_PREFIXES_COUNT + 3)) {
+            String schemaName = "test_i_s_columns_schema" + randomNameSuffix();
+            assertUpdate("CREATE SCHEMA " + schemaName);
+            Session session = Session.builder(getSession())
+                    .setSchema(schemaName)
+                    .build();
+
+            for (int i = 0; i < tables; i++) {
```

Comment:
Is `io.trino.plugin.deltalake.TestDeltaLakeFileOperations#testInformationSchemaColumns`  and `io.trino.plugin.deltalake.metastore.TestDeltaLakeMetastoreAccessOperations#testSystemMetadataTableComments` going to be flaky now?

(there are table metadata updates enqued now when doing `information_schema.columns` and `system.metadata.table_comments` operations)

Reply:
It should not be flaky after #23602 which changed the scheduler's interval to 30m from 1s.